.github/workflows/python-ci.yml

```diff
@@ -17,20 +17,19 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
-      env:
-        PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
       run: |
         sudo apt-get update
         sudo apt-get install cdo
-        pip install .[complete]
-        pip install .[test]
+        pip install -U pip pipenv
+        pipenv install rpy2==3.3.6
+        pipenv install --dev
```

Comment:
Since these packages have been added into the `pipfiles` we don't have to install them using the `.[scenario]` tags. You can simply use `pipenv install`.